src/MongoDB.Driver.Core/Core/Authentication/CryptographyHelpers.cs

```diff
@@ -0,0 +1,175 @@
+﻿/* Original work:
```

Comment:
From the CoreFX project.

Reply:
Only some very minor formatting changes, a namespace change, and some changes to use expression bodies for properties.

Reply:
Reverted changes to properties and formatting.
Original name of the class was `Helpers` under System.Cryptography.Internal.
I've vendored it it as "CryptographyHelpers" under `MongoDB.Driver.Core.Authentication.Vendored.`
I also imported a string that was defined in a resource file.